harvest/time.go

```diff
@@ -0,0 +1,39 @@
+package harvest
+
+import (
+	"net/url"
+	"strings"
+	"time"
+)
+
+type Time struct {
+	time.Time
+}
+
+func (t Time) String() string {
+	return strings.ToLower(t.Time.Format(time.Kitchen))
+}
+
+func (t *Time) UnmarshalJSON(data []byte) (err error) {
+	str := string(data)
+	str = strings.Trim(str, "\"")
+	str = strings.ToUpper(str)
+	resp, err := time.Parse(time.Kitchen, str)
+	if err != nil {
+		return err
+	}
+
+	now := time.Now()
+	(*t).Time = time.Date(now.Year(), now.Month(), now.Day(), resp.Hour(), resp.Minute(), 0, 0, now.Location())
```

Comment:
Q: have you tested this when you selected entries from last month? Do you get the startedTime back from the API?
If so, your date will be wrongly set to the current date.